notebooks/rendered/bigquery-query-magic.md

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
No magic registration steps? Possibly not needed, depending on where these notebooks are run.

Reply:
Added some installation steps in a readme